CHANGELOG.md

```diff
@@ -32,6 +32,9 @@ Entries marked with a ︎⚠️ symbol require particular attention during upgra
 
 % ### Deprecations and removals
 
+* Removed the docker images implemenation, deployment and documentation from the
+  Eradiate repository
```

Comment:
Let's reference this PR for more information:
```suggestion
* Removed the docker images implemenation, deployment and documentation from the
  Eradiate repository ({ghpr}`322`).
```

Reply:
And voilà